hack/verify-yamllint.sh

```diff
@@ -29,12 +29,11 @@ if [[ "${deployDirNum}" != "${helmDirNum}" ]]; then
   exit 1
 fi
 
-for path in "deploy/*.yaml" "deploy/example/*.yaml" "deploy/example/metrics/*.yaml" "deploy/example/snapshot/*.yaml" "deploy/example/cloning/*.yaml" "deploy/example/rawblock/*.yaml" "deploy/example/windows/*.yaml" "deploy/example/sharedisk/*.yaml" "docs/node-shutdown-recovery/*.yaml"
```

Comment:
why remove `docs/node-shutdown-recovery/*.yaml`?

Reply:
In my test, it failed because path `docs/node-shutdown-recovery` does not exist. It should be `docs/known-issues/node-shutdown-recovery/*.yaml`. I will add it back in my next PR.